templates/Default/engine/Default/planet_construction.php

```diff
@@ -1,4 +1,4 @@
-<p>You are currently building: <?php
+<p><span name="currently_building">You are currently building: <?php
```

Comment:
The added `<span>` does not introduce a refresh area. The entire middle panel is being updated even without this change (which I believe is desirable and the original intent for this page).

What this change _does_ do is fix a weird issue with the "Cancel" button jumping to a different location on the page after an ajax update. Specifically, the DOM parsing of that one particular `<p>...</p>` on this page is broken. As you can see from https://github.com/smrealms/smr/pull/99#issuecomment-318519929, the `convertHtmlToAjaxXml` code that is only run to determine how the page looks after an ajax update puts the `</p>` in the wrong place! Adding the `<span>` forces it to keep the `</p>` in the right place. (If you'd like to see it visually, I can take screenshots with and without the span.)

I don't understand why the DOM parsing is buggy, but it is. This change only circumvents the bug, so if you have some idea how to fix it without introducing a `<span>`, I'm all for it! But if it's just a choice between correct ajax updates and an extra html element vs. incorrect ajax updates, I'd take the former any day. :)